mtp_bank_admin/templates/bank_admin/dashboard.html

```diff
@@ -10,12 +10,16 @@
 </ul>
 <p/>
 {% endif %}
+{% if perms.transaction.view_bank_details_transaction %}
 <a href="{% url "bank_admin:download_refund_file" %}">
 	{% trans "Download Refunds" %}
 </a>
+<br/>
+{% endif %}
 <a href="{% url "bank_admin:download_adi_payment_file" %}">
 	{% trans "Download ADI Payment Reconciliation File" %}
 </a>
+<br/>
```

Comment:
I might be tempted to wrap these in `p` rather than `br`, simply because `br` is evil.